src/components/task-plan/homework/exercises.cjsx

```diff
@@ -33,6 +33,9 @@ ExerciseCardMixin =
   renderExercise: ->
     content = @props.exercise.content
     question = content.questions[0]
+    question.answers = _(question.answers).chain()
+      .sortBy('id')
+      .value()
     renderedAnswers = _.map(question.answers, @renderAnswers)
```

Comment:
Nice, `sortBy` using a property, instead of a function.  I think that's much clearer that way.

The `_(` in underscore is a shorthand for the chain method though.  So you don't need to append the `.chain`  http://underscorejs.org/#chaining

It also might make sense to also include the `_.map` below into the chain though. i.e.
```coffee
  renderedAnswers = _(question.answers)
    .sortBy('id')
    .map(question.answers, @renderAnswers)
    .value()
```